osbs/api.py

```diff
@@ -671,22 +672,41 @@ def create_orchestrator_build(self, **kwargs):
             raise
 
     @osbsapi
-    def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missing=False):
+    def get_build_logs(self, build_id, follow=False, build_json=None, wait_if_missing=False,
+                       decode=False):
         """
         provide logs from build
 
-        NOTE: since there is no way to determine the text encoding in use
-        the bytes are returned exactly as they came from the container.
+        NOTE: Since atomic-reactor 1.6.25, logs are always in UTF-8, so if
+        asked to decode, we assume that is the encoding in use. Otherwise, we
+        return the bytes exactly as they came from the container.
 
         :param build_id: str
         :param follow: bool, fetch logs as they come?
         :param build_json: dict, to save one get-build query
         :param wait_if_missing: bool, if build doesn't exist, wait
-        :return: None, bytes, or iterable of bytes
+        :param decode: bool, whether or not to decode logs as utf-8
+        :return: None, iterable of bytes or iterable of strs
         """
-        return self.os.logs(build_id, follow=follow, build_json=build_json,
+        logs = self.os.logs(build_id, follow=follow, build_json=build_json,
                             wait_if_missing=wait_if_missing)
 
+        if logs is None:
+            yield None
```

Comment:
Good catch. Changed that to a return instead of yield None, had originally tried return None, which at least python 2.x doesn't like.